<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tasks_Window</class>
 <widget class="QMainWindow" name="Tasks_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks_Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="Groups_listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>220</width>
      <height>409</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="GroupName_lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>142</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddGroup_Button">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>420</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Tasks_listWidget">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>0</y>
      <width>347</width>
      <height>373</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="Task_groupBox">
    <property name="geometry">
     <rect>
      <x>587</x>
      <y>0</y>
      <width>221</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="TaskName_lineEdit">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>20</y>
       <width>148</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="Complete_Box">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>24</y>
       <width>15</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="TaskNotes_textEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>129</y>
       <width>168</width>
       <height>119</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CompleteDate_Button">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>254</y>
       <width>168</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата выполнения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Remind_Button">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>283</y>
       <width>168</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Напоминание</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cycle_Button">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>312</y>
       <width>168</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Повтор</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Delete_Button">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>415</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Add_groupBox">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>384</y>
      <width>347</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="AddTask_Button">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>0</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameTask_lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>267</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AddButtons_groupBox">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>412</y>
      <width>347</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="CompleteDate1_Button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>112</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата выполнения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Remind1_Button">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>0</y>
       <width>112</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Напоминание</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Delete1_Button">
     <property name="geometry">
      <rect>
       <x>286</x>
       <y>0</y>
       <width>58</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cycle1_Button">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>0</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Повтор</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
